include/ttmlir/Dialect/TTIR/Transforms/Passes.td

```diff
@@ -108,6 +108,14 @@ def TTIRAllocate: Pass<"ttir-allocate", "::mlir::ModuleOp"> {
   }];
 }
 
+def TTIRDecompose: Pass<"ttir-decompose", "::mlir::ModuleOp"> {
```

Comment:
This is no longer needed, right?

Reply:
Yeah, I'm still testing out the changes so I will remove everything unnecessary once finished